package.json

```diff
@@ -9,7 +9,7 @@
     "@fortawesome/free-brands-svg-icons": "^5.12.0",
     "@fortawesome/react-fontawesome": "^0.1.8",
     "@loadable/component": "^5.12.0",
-    "@tippy.js/react": "^3.1.1",
+    "@tippyjs/react": "^4.2.6",
```

Comment:
Package was deprecated, which is why CSS styling wasn't showing up for some components using this. See: https://www.npmjs.com/package/@tippy.js/react